src/tools/ApiChecker/ApiChecker.csproj

```diff
@@ -1,20 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
-
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp3.1</TargetFramework>
-    <ReleaseVersion>0.0.0</ReleaseVersion>
   </PropertyGroup>
-
+  <Import Project="../../Common/PackageInfo.props" />
```

Comment:
We could also add `Dependencies.props` so we analyzers.

Reply:
Yeah, good point. It means changing the `Dependencies.props` though. But that's fine, I think.